src/Api/Repositories.php

```diff
@@ -268,6 +268,23 @@ public function commitRefs($project_id, string $sha, array $parameters = [])
         );
     }
 
+    /**
+     * @param int|string $project_id
+     * @param string     $sha
+     * @param array      $parameters
+     *
+     * @return mixed
+     */
+    public function commitMergeRequests($project_id, string $sha, array $parameters = [])
```

Comment:
We need param documentation and validation for the supported parameters please.

Reply:
I've removed the "parameters" parameter as this api endpoint does not seem to support any parameters.